.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Test tephi
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/cache@v2
+      with:
+        key: tox
```

Comment:
Do you think you ought to hash `tox.ini` and `ci/requirements/dev.yml` too? AFAICT this will only cache once then re-use that cache for all of time?

Reply:
Good idea, I've added a more complex cache key